package.json

```diff
@@ -59,9 +59,9 @@
     "@types/shallowequal": "^1.1.0",
     "@types/yup": "^0.26.3",
     "@welldone-software/why-did-you-render": "^3.2.3",
-    "card-validator": "^6.1.0",
+    "card-validator": "^8.1.1",
```

Comment:
Do you need to do these upgrades as a part of this PR? It might be better to do it separately to reduce the risk?

Reply:
I have reversed the upgrade, only installed `@types/enzyme-adapter-react-16`.